### ISSUE TITLE
Investigate Rust-WASM bindings as a precursor towards selecting a methodology and toolchain modules

### ISSUE THREAD

**Issue by amilumibe** [walletd/walletd#54]

**Comment by amilumibe** [comment]
link to issue with rust-secp256k1 wasm on macOS: https://github.com/rust-bitcoin/rust-secp256k1/pull/254

**Comment by amilumibe** [comment]
PR draft seems to build walletd wasm ok: https://github.com/walletd/walletd-wasm/pull/1

**Comment by devinpearson** [comment]
Putting WASM builds on hold for now as we tackle more features for the coins we have. This is blocked by some of the libraries we use not being compatible with WASM builds
